yb_migrate/src/migration/postgres.go

```diff
@@ -243,7 +251,7 @@ func PgDumpExportDataOffline(ctx context.Context, source *srcdb.Source, exportDi
 	// Wait for pg_dump to complete before renaming of data files.
 	err = proc.Wait()
 	if err != nil {
-		utils.PrintAndLog("pg_dump failed to export data with error: %v\n%s", err, buf.String())
+		utils.PrintAndLog("pg_dump failed to export data with error: %v\n%s", err, errbuf.String())
```

Comment:
The `errbuf` contents should only go to the log.

Point the user to the log file in the message sent on the STDOUT.